api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java

```diff
@@ -66,10 +63,6 @@ public class PlatformTransactionData {
   @SerializedName("external_transaction_id")
   String externalTransactionId;
 
-  // TODO: review the use case of the custodialTransactionId
-  @SerializedName("custodial_transaction_id")
-  String custodialTransactionId;
```

Comment:
I don't see this field being removed in a migration file. Did it not exist in the first place?

Reply:
It's never been a part of initial SQL script